tools/bazel/vitest/coverage_runner.sh

```
@@ -0,0 +1,267 @@
+if [[ -n "${coverage_report_destination}" ]]; then
+  env -u COVERAGE -u COVERAGE_DIR -u COVERAGE_MANIFEST -u COVERAGE_OUTPUT_FILE \
+    "${vitest_binary}" "${args[@]}"
+else
+  "${vitest_binary}" "${args[@]}"
+fi
+
+if [[ -n "${coverage_report_destination}" ]]; then
+  readonly lcov_report="${reports_dir}/lcov.info"
+  readonly rewritten_lcov_report="${reports_dir}/lcov.rewritten.raw.info"
+  readonly normalized_lcov_report="${reports_dir}/lcov.rewritten.info"
+  readonly baseline_coverage_report="$(dirname "${coverage_report_destination}")/../baseline_coverage.dat"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)